frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironment.tsx

```diff
@@ -131,6 +132,11 @@ const FeatureOverviewEnvironment = ({
     const featureEnvironment = feature?.environments.find(
         (featureEnvironment) => featureEnvironment.name === env.name,
     );
+    const resourceLimitsEnabled = useUiFlag('resourceLimits');
+    const limitReached =
+        resourceLimitsEnabled &&
+        Array.isArray(featureEnvironment?.strategies) &&
+        featureEnvironment?.strategies.length >= 30;
```

Comment:
I think it is a terrible idea to hard code these values here. I think at least they should be moved to a central configuration sections sot that we keep controls of UI limits. 

I also find it a bit strange to limit this in the UI but not via the API.

Reply:
This PR is still WIP and eventually we'll have one value shared between backend and frontend. I'm currently hardcoding those so that I can finish writing UI tests